### PR TITLE
Optivorbis lossless compression for the sound folder

### PR DESCRIPTION
**Pull request by grungussuss**
## About The Pull Request
used:
 https://github.com/OptiVorbis/OptiVorbis
and a script provided by gpeckman to run it through the entire sound folder:
https://discord.com/channels/326822144233439242/326831214667235328/1294688179919523862

### Comparison:
before: 
image
after:
image
with a total reduction of 9%

## Why It's Good For The Game
takes up less space - downloads faster!
## Changelog
:cl: grungussuss and gpeckman
sound: the sound folder has been optimized to take up less space
/:cl:

### CODE REVIEW

**Comment by Sadboysuss** [comment]
tested by loading in and floating around as a ghost a bit

File: sound/ambience/maintenance/attribution.txt [hunk 1]
```diff
@@ -1,2 +1 @@
-ambispace4.ogg is Sipping Bird, by Capsandi. It has been licensed under CC-BY 3.0 license.
-ambimaint6.ogg and ambimaint7.ogg, created by Capsandi are licensed under CC-BY 3.0 license.
+magma.ogg - Hot spring.Seething and bubbles(2lrs,mltprcssng).wav by newlocknew -- https://freesound.org/s/581417/ -- License: Attribution 4.0 and wind 5 by ZIP.Creates -- https://freesound.org/s/726316/ -- License: Creative Commons 0
```
Comment (JohnFulpWillard) [inline sound/ambience/maintenance/attribution.txt:1]: im sure you've explained somewhere but can you explain on github for looking back in the future why you're removing attribution for ambispace4 and ambimaint6?
Also,
1. ``magma.ogg`` is not in this folder
2. this seems like a copy paste of ``sound/ambience/attribution.txt``
3. What's 'wind 5'? I can't find a file called that, but listening in to both sounds individually and the magma sound we have in game, sounds like you merged the two into one sound, can you specify that it is the case, in the proper file, as per the requirements

image

Comment (Sadboysuss) [reply sound/ambience/maintenance/attribution.txt:1]: mistakes were made, I'll fix it. the license fix should probably be in anothe pr tho

**Comment by Sadboysuss** [comment]
> fwiw I made a simple tool to recursively run optivorbis in a folder: https://github.com/Absolucy/meowvorbis
> 
> likely faster than the script as it's multithreaded and uses optivorbis as a library, rather than running the executable for every file.

yeah that's what we did too

**Comment by Sadboysuss** [comment]
I think it's good to merge

**Pull request merged**